src/SignalRServiceExtension/Bindings/SignalRConnectionInfoV2.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    public class SignalRConnectionInfoV2
```

Comment:
Shall we add a separate input binding that only contains access token information, so:
1. You don't need a V2 class
2. Seems in your sample there're some cases that you only need the access token information.